radis/io/hitran.py

```diff
@@ -951,9 +956,62 @@ def parse_global_quanta(df, mol):
             "Unknown class for molecule {0}. Cant parse global quanta".format(mol)
         )
 
+    df = reduce_mem_usage(df)
+
     return df
 
 
+def reduce_mem_usage(props):
```

Comment:
do you have any idea of how long it takes to run this function ?

Also ; even if the general-purpose function is great for testing, I think it's very dangerous to auto-convert quantities which may differ from one database to another.

I'd rather hardcode the column names.

Reply:
I will dig into this and create a fresh PR.